cluster/saltbase/salt/calico/calico-policy-controller.manifest

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: calico-policy-controller
+  namespace: kube-system
+  labels:
+    version: latest
```

Comment:
maybe add 

`kubernetes.io/cluster-service: "true"`

and 

`k8s-app: calico-policy`